### PR TITLE
Added build instructions for FreeBSD

### PR DESCRIPTION
**Pull request by SamuelGogarty**
Adds documentation on how to build and install Azerothcore on FreeBSD

### Description
Covers how a user can install Azerothcore on FreeBSD, covers installation of build and runtime depend's, cmake configuration and compilation and installation of the project.
### Related Issue
https://github.com/azerothcore/azerothcore-wotlk/pull/9568

### CODE REVIEW

**Comment by SamuelGogarty** [comment]
@Kitzunu can you please review this? thanks.
I would prefer if this was merged in only after https://github.com/azerothcore/azerothcore-wotlk/pull/9568 is merged.

File: docs/FreeBSD.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# FreeBSD Install
```
Comment (Kitzunu) [inline docs/FreeBSD.md:1]: keep file names only lower case.

the first header should match the file name. (The header can be capitalized as wished)

File: docs/FreeBSD.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+make -j<number of cores>
+```
+
+Install Azerothcore
+
+```
+make install
+```
+
+
```
Comment (Kitzunu) [inline docs/FreeBSD.md:55]:
```suggestion
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: docs/FreeBSD.md (hunk 1 above)
Comment (Kitzunu) [reply docs/FreeBSD.md:1]: https://www.azerothcore.org/wiki/wiki-standards

**Comment by SamuelGogarty** [comment]
@Kitzunu file renamed and extra whitespace removed

File: docs/freebsd.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# FreeBSD Install
```
Comment (Kitzunu) [inline docs/freebsd.md:1]: the first header should match the file name. (The header can be capitalized as wished)

**Comment by SamuelGogarty** [comment]
@Kitzunu removed 'Install' from header to conform to standards

**Review by Helias — APPROVED** [review]

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**